packages/openapi-typescript/test/cli.test.ts

```diff
@@ -131,5 +129,16 @@ describe("CLI", () => {
         ).toMatchFileSnapshot(fileURLToPath(new URL("./examples/simple-example.ts", root)));
       }
     });
+
+    test("lint error", async () => {
+      const cwd = new URL("./fixtures/redocly-lint-error", import.meta.url);
+
+      try {
+        await execa("../../../bin/cli.js", { cwd: fileURLToPath(cwd) });
+        throw new Error("Linting should have thrown an error");
+      } catch (err) {
+        expect(stripAnsi(String(err))).toMatch(/✘ {2}Servers must be present/);
```

Comment:
Not exhaustive, but strikes a good balance between asserting that the linting throw an error with a specific message, vs tying tests too-heavily to Redocly’s specific output that could change.